src/plugin/panel/meson.build

```diff
@@ -78,11 +79,11 @@ libpanelplugin_gir = gnome.generate_gir(
     ],
     symbol_prefix: 'budgie',
     identifier_prefix: 'Budgie',
+    header: 'plugin.h',
     install: true,
     extra_args: [
-        '--warn-all',
```

Comment:
Why was this removed?

Reply:
Good question, will have to double check on that one.

Reply:
Resolved in d9e2bdef7d3d8bd2cda76b281dc290bfcd305adb